src/app/zap-templates/zcl/data-model/chip/onoff-cluster.xml

```diff
@@ -52,7 +60,7 @@ limitations under the License.
     <attribute side="server" code="0x4000" define="GLOBAL_SCENE_CONTROL" type="boolean" default="1"                 optional="true">GlobalSceneControl</attribute>
     <attribute side="server" code="0x4001" define="ON_TIME"              type="int16u"  default="0" writable="true" optional="true">OnTime</attribute>
     <attribute side="server" code="0x4002" define="OFF_WAIT_TIME"        type="int16u"  default="0" writable="true" optional="true">OffWaitTime</attribute>
-    <attribute side="server" code="0x4003" define="START_UP_ON_OFF"      type="enum8"               writable="true" optional="true">StartUpOnOff</attribute>
+    <attribute side="server" code="0x4003" define="START_UP_ON_OFF"      type="DlStartUpOnOff" min="0" max="2" writable="true" optional="true">StartUpOnOff</attribute>
```

Comment:
This is going to conflict with https://github.com/project-chip/connectedhomeip/pull/16377, which should be a higher priority because it's an actual spec-compliance bug.

Reply:
As #16377 made it into the master, this PR has been rebased on master to incorporate its changes.